src/spelltranslator.rs

```diff
@@ -613,21 +613,23 @@ fn parse_about_line(equation: &str) -> Result<Vec<u64>, &'static str>{
             opcodes.extend(match match values.strip_prefix('[')
             .and_then(|x| x.strip_suffix(']'))
             .ok_or_else(|| "Invalid parameters: should be a list and have \"[\" \"]\"")?
-            .split(',')
+            
+            .split(',').collect::<Vec<&str>>()[..] {
```

Comment:
This collect just seems like it's adding more overhead to me, but not so much that I won't accept it.